docs/smart-contracts/guides/conventions/install-deploy-contract-with-code.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Install and deploy a smart contract with code
+hide_table_of_contents: true
+draft: true
+---
+
+<head>
+  <title>Install and deploy a smart contract with code</title>
+  <meta charSet="utf-8" />
+  <meta
+    property="og:title"
+    content="Install and deploy a smart contract with code"
+  />
+  <meta
+    property="og:description"
+    content="This guide will walk you through the process of installing and deploying a smart contract using js-stellar-sdk. We will cover the setup of a sample Rust contract, creating a Node.js project to manage the deployment, and finally, installing the wasm of the contract and deploying it to the network"
+  />
+</head>
+
+This guide will walk you through the process of installing and deploying a smart contract using [js-stellar-sdk](https://github.com/stellar/js-stellar-sdk). We will cover the setup of a sample Rust contract, creating a Node.js project to manage the deployment, and finally, installing the wasm of the contract and deploying it to the network
+
+# Prerequisites
+
+Before you begin, ensure you have the following installed:
+
+1. [Rust](https://www.rust-lang.org/) and Cargo (for compiling smart contracts)
+2. [Node.js](https://nodejs.org/en) and npm (for running JavaScript deployment scripts)
+3. [Soroban CLI](../../getting-started/setup.mdx#install-the-soroban-cli)
+
+# Initialize a sample Rust Contract
+
+```bash
+mkdir ~/hello-world
+cd hello-world
+soroban contract init
```

Comment:
```
cd: no such file or directory: hello-world
error: the following required arguments were not provided:
  <PROJECT_PATH>

Usage: soroban contract init <PROJECT_PATH>

For more information, try '--help'.
```

Could you please step through the tutorial and let me know when it has been updated? Thank you!